projects/core/src/i18n/i18next/i18next-translation.service.ts

```diff
@@ -73,16 +91,23 @@ export class I18nextTranslationService implements TranslationService {
 
   /**
    * Returns a fallback value in case when the given key is missing
-   * @param key
+   * @param keys
    */
-  protected getFallbackValue(key: string): string {
-    return isDevMode() ? `[${key}]` : this.NON_BREAKING_SPACE;
+  protected getFallbackValue(keys: string[]): string {
+    return isDevMode()
+      ? keys.map((key) => `[${key}]`).join(this.NON_BREAKING_SPACE)
```

Comment:
What about simply separate keys with `,`  _inside_ one pair of brackets? By example:
```
[common:helloWorld, common:hello, common:hi]
```
instead of
```
[common:helloWorld] [common:hello] [common:hi]
                   ^ --- nbsp --- ^
```